src/others/contexts/file-downloader.tsx

```diff
@@ -98,15 +98,12 @@ export const FileDownloaderContextProvider: FunctionComponent = ({ children }) =
       const csvRow = [requestDate];
 
       const categoryToAmount: { [id: string]: number } = {};
+      uniqueCategories.forEach((category) => { categoryToAmount[category] = 0; });
       cityRequests.forEach((request) => {
         categoryToAmount[request.category] = request.amount;
       });
       uniqueCategories.forEach((category) => {
-        if (category in categoryToAmount) {
-          csvRow.push(categoryToAmount[category] + "");
-        } else {
-          csvRow.push("0");
-        }
+        csvRow.push(categoryToAmount[category] + "");
```

Comment:
**suggestion/non-blocking**

I'd argue that `String(categoryToAmount[category])` is more readable, but I'll leave it to you to decide. :)

Reply:
done